host-controller/src/main/java/org/jboss/as/domain/controller/operations/ProfileCloneHandler.java

```diff
@@ -53,21 +59,46 @@
  */
 public class ProfileCloneHandler implements OperationStepHandler {
 
-    public static final ProfileCloneHandler INSTANCE = new ProfileCloneHandler();
     public static final OperationDefinition DEFINITION = new SimpleOperationDefinitionBuilder(CLONE, DomainResolver.getResolver(PROFILE, false))
             .addAccessConstraint(SensitiveTargetAccessConstraintDefinition.READ_WHOLE_CONFIG)
             .addParameter(SimpleAttributeDefinitionBuilder.create(TO_PROFILE, ModelType.STRING).build())
             // .setPrivateEntry()
             .build();
 
+    private final LocalHostControllerInfo hostInfo;
+    private final IgnoredDomainResourceRegistry ignoredDomainResourceRegistry;
+
+    public ProfileCloneHandler(LocalHostControllerInfo hostInfo, IgnoredDomainResourceRegistry ignoredDomainResourceRegistry) {
+        this.hostInfo = hostInfo;
+        this.ignoredDomainResourceRegistry = ignoredDomainResourceRegistry;
+    }
 
     @Override
     public void execute(final OperationContext context, final ModelNode operation) throws OperationFailedException {
-
         final String profileName = PathAddress.pathAddress(operation.require(OP_ADDR)).getLastElement().getValue();
         final String newProfile = operation.require(TO_PROFILE).asString();
         final String operationName = GenericModelDescribeOperationHandler.DEFINITION.getName();
         final PathAddress address = PathAddress.pathAddress(PathElement.pathElement(PROFILE, profileName));
+        final PathAddress newPA = PathAddress.pathAddress(PROFILE, newProfile);
+
+        if (!hostInfo.isMasterDomainController()) {
+            if (ignoredDomainResourceRegistry.isResourceExcluded(address) || ignoredDomainResourceRegistry.isResourceExcluded(newPA)) {
+                return;
+            }
+            Resource root = context.readResourceFromRoot(PathAddress.EMPTY_ADDRESS);
+            Resource host = root.getChildren(HOST).iterator().next();
+            boolean ignoreUnusedConfiguration =
+                    RemoteDomainControllerAddHandler.IGNORE_UNUSED_CONFIG.resolveModelAttribute(context,
+                            host.getModel().get(
+                                    HostResourceDefinition.DOMAIN_CONTROLLER.getName(),
+                                    HostResourceDefinition.DC_REMOTE.getName())).asBoolean();
+
+            if (ignoreUnusedConfiguration) {
+                //Since the cloned profile is a new one, we don't need to do anything. It is guaranteed to be unused,
+                //and thus should be ignored
+                return;
+            }
```

Comment:
I'm going to send up a new PR changing this to just call LocalHostControllerInfo.isRemoteDomainControllerIgnoreUnaffectedConfiguration() as that will be necessary for this to work properly once the WFCORE-310 work comes in. There will no longer be a default value for IGNORE_UNUSED_CONFIG; the behavior when undefined will depend on whether --backup is used.